backend/makefiles/batch.mk

```diff
@@ -1,13 +1,13 @@
 batch.seed_all: ## Populate all data sequentially
-	batch.seed_tickers
-	batch.seed_ohlc
-	batch.seed_covariance
+	make batch.seed_tickers
```

Comment:
you can just add the recipes as prerequisites.

ie. batch.seed_all: batch.seed_tickers batch.seed_ohlc...

https://www.gnu.org/software/make/manual/html_node/Rule-Introduction.html#Rule-Introduction